### PR TITLE
[AJUN-524] Update bajun-parachain to polkadot-sdk v1.13.0 

### PR DESCRIPTION
**Pull request by DidacSF**
## Description

* Update deps to `1.13.0`

## Type of changes

- [x] `build`: Changes that affect the build system or external dependencies (eg, Cargo, Docker)
- [ ] `ci`: Changes to CI configuration
- [ ] `docs`: Changes to documentation only
- [ ] `feat`: Changes to add a new feature
- [ ] `fix`: Changes to fix a bug
- [ ] `refactor`: Changes that do not alter functionality
- [ ] `style`: Changes to format the code
- [ ] `test`: Changes to add missing tests or correct existing tests

## Checklist

- [ ] Tests for the changes have been added
- [ ] Necessary documentation is added (if appropriate)
- [ ] Formatted with `cargo fmt --all`
- [ ] Linted with `cargo clippy --all-features --all-targets`
- [ ] Tested with `cargo test --workspace --all-features --all-targets`

### CODE REVIEW

File: runtime/ajuna/src/xcm_config.rs [hunk 1]
```diff
@@ -390,28 +390,31 @@ pub type LocalOriginToLocation = SignedToAccountId32<RuntimeOrigin, AccountId, R
...
+	type Currency = Balances;
+	type CurrencyMatcher = ();
 	type SendXcmOrigin = EnsureXcmOrigin<RuntimeOrigin, ()>; // Prohibit sending arbitrary XCMs from users of this chain
 	type XcmRouter = XcmRouter;
-	type ExecuteXcmOrigin = EnsureXcmOrigin<RuntimeOrigin, LocalOriginToLocation>; // Allow any local origin in XCM execution.
-	type XcmExecuteFilter = Nothing; // Disable generic XCM execution. This does not affect Teleport or Reserve Transfer.
+	type ExecuteXcmOrigin = EnsureXcmOrigin<RuntimeOrigin, LocalOriginToLocation>;
+	// Allow any local origin in XCM execution.
+	type XcmExecuteFilter = Nothing;
+	// Disable generic XCM execution. This does not affect Teleport or Reserve Transfer.
```
Comment (clangenb) [inline runtime/ajuna/src/xcm_config.rs:400]: The formatting put the comments on the wrong line, they describe the item above now. Can you move them on top of the item they describe?

File: Cargo.toml
Comment (clangenb) [inline Cargo.toml:1]: So you decided to no longer give a damn about the custom formatting? Fine with me, just not sure if this is intentional.

**Review by clangenb — COMMENTED** [review]
Some formatting remarks, but it looks good otherwise!

File: Cargo.toml
Comment (DidacSF) [reply Cargo.toml:1]: Not intentional... it's my damn CLion how not matter how many times I turn its auto-formatting off, keeps reenabling it all the damn time

File: runtime/ajuna/src/xcm_config.rs (hunk 1 above)
Comment (DidacSF) [reply runtime/ajuna/src/xcm_config.rs]: Adjusted formatting!

File: runtime/ajuna/src/xcm_config.rs [hunk 2]
```diff
@@ -425,11 +428,32 @@ impl cumulus_pallet_xcmp_queue::Config for Runtime {
...
+
+impl ListChannelInfos for AjunaActiveXCMChannels {
+	fn outgoing_channels() -> Vec<ParaId> {
+		vec![ASSET_HUB_PARA_ID, HYDRA_DX_PARA_ID]
+	}
+}
+
+impl cumulus_pallet_xcmp_queue::migration::v5::V5Config for Runtime {
+	type ChannelList = AjunaActiveXCMChannels;
 }
```
Comment (clangenb) [inline runtime/ajuna/src/xcm_config.rs:457]: Unfortunately, I realized that we can't do it like this, as the actual runtime migration on Paseo will fail, not only the try runtime one.


I will check the outgoing connections manually myself for sanity, and then write a custom migration that is a no-op and simply updates the storage version of the xcm queue.

**Review by clangenb — APPROVED** [review]
Looks good to me, we can fix the migration later. I suggest that we merge this so that you can continue your work!

File: runtime/ajuna/src/xcm_config.rs [hunk 3]
```diff
@@ -390,28 +390,31 @@ pub type LocalOriginToLocation = SignedToAccountId32<RuntimeOrigin, AccountId, R
...
-	type XcmExecuteFilter = Nothing; // Disable generic XCM execution. This does not affect Teleport or Reserve Transfer.
+	// Allow any local origin in XCM execution.
+	type ExecuteXcmOrigin = EnsureXcmOrigin<RuntimeOrigin, LocalOriginToLocation>;
+	// Disable generic XCM execution. This does not affect Teleport or Reserve Transfer.
+	type XcmExecuteFilter = Nothing;
 	type XcmExecutor = XcmExecutor<XcmConfig>;
 	type XcmTeleportFilter = OnlyTeleportNative;
-	type XcmReserveTransferFilter = Everything; // Transfer are allowed
+	type XcmReserveTransferFilter = Everything;
+	// Transfer are allowed
```
Comment (clangenb) [inline runtime/ajuna/src/xcm_config.rs:404]: Oh, I forgot this one, this comment is also misplaced.

**Pull request merged**